en/engine/animation/skeletal-animation.md

```diff
@@ -0,0 +1,73 @@
+
+## Pre-baked Skeletal Animation System
+
+The dominant purpose of this system is performance, and some sacrifices in expressiveness are also considered acceptable. __Cocos Creator 3D__ has made many low-level optimizations in a targeted manner. The current runtime process is roughly as follows:
+  * All animation data will be pre-sampled in advance according to the specified frame rate and baked onto a global-managed joint texture atlas.
+  * Depending on whether the operating platform supports floating-point textures, the corresponding texture format will be RGBA32F, or automatically fallback to RGBA8 if not available (The rendering results should be identical, it is only the fail-safe approach for really low-end devices, and shouldn't be of any concern for game developers).
+  * Each __Skeletal Animation Component__ (`SkeletalAnimationComponent`) is responsible for maintaining the current playback progress, stored in the form of UBO (a `vec4`).
+  * Each skinning model component (`SkinningModelComponent`) holds a pre-baked skinning model class (`BakedSkinningModel`). Based on the bounding box information pre-baked in the same way to do frustum culling, update the UBO, and get the current data from the texture atlas on the GPU to complete the skinning.
+
+## Real-time calculated Skeletal Animation System
+
+The dominant purpose of this system is expressiveness, ensuring the correct display of all details, and complete program control capabilities.
+
+The current runtime process is roughly as follows:
+  * All animation data are calculated dynamically according to the current global time;
+  * Animation data will be output to the skeleton node tree of the scene;
+  * Users and any other system can affect the skin effect by manipulating this bone node tree;
+  * Each skinning model component (`SkinningModelComponent`) holds a common skinning model class (`SkinningModel`). Extract the transformation data from joint node tree, do frustum culling, upload the complete bone transformation information of the current frame to UBO, and complete the skinning in the GPU.
+
+This provides the fundamental support for all the following functions:
+  * blendshape support
+  * Mixing and masking of any number of animation clips
+  * Inverse kinematics, secondary physics
+  * Explicit procedural control over any joint tranformations
+
+## Selection and best practice of two systems
+
+After importing all model resources, all prefabs use the pre-baking system by default to achieve the best performance. It is recommended that you only use the real-time computing system if you clearly feel that the performance of the pre-baking system cannot reach the standard. Although the two systems can be switched seamlessly at runtime, try to do this as less frequent as possible, because each switch involves the reconstruction of the underlying rendering data.
```

Comment:
>  It is recommended that you only use the real-time computing system

It is recommended that you only use the **real-time calculated** system

It is important to keep the terminology references consistent through out the doc, although the particular naming of the systems could be reconsidered later.